syntax = "proto3";
package common;
option go_package = "/proto";

message Empty {}

message JwtPayload {
  string userId = 1;
  string email = 2;
}

message UserProfile {
  string id = 1;
  string email = 2;
}

message Pagination {
  int32 page = 1;
  int32 perPage = 2;
}

message PaginationMeta {
  int32 total = 1;
}

message ById {
  string id = 1;
}
